pkg/kube/pod.go

```diff
@@ -31,6 +33,9 @@ import (
 	"github.com/kanisterio/kanister/pkg/poll"
 )
 
+// podReadyWaitTimeout is the time to wait for pod to be ready
+const podReadyWaitTimeout = 5 * time.Minute // Q: Is 5 minutes sufficient?
```

Comment:
Did you mean to make this a PR question or something to follow up on later?
If the later, then file a ticket, otherwise:

```suggestion
const podReadyWaitTimeout = 5 * time.Minute
```

Reply:
@julio-lopez yes thanks, I meant to make it a PR question. 

@tdmanv @julio-lopez would love to get your input on this Q.

Reply:
I do not know what a good value would be, that seems long enough, but not sure what other implications it may have. @tdmanv should be able to provide better guidance here.

Reply:
I also don't have a good sense here, but 5 minutes sounds as good as anything. Let's include 5 minutes in the error we return so we know we what this is set to.